src/core/service.c

```diff
@@ -453,6 +453,7 @@ static int service_add_fd_store(Service *s, int fd, const char *name, bool poll)
 
         fs->fd = fd;
         fs->service = s;
+        fs->poll = poll;
```

Comment:
please name this differently, here too. i mean, it's the worst kind of namespace clash even: the field is called exactly as the function it is supposed to control...

Reply:
Naming conventions like these are entirely a matter of taste. I have no problem renaming it to `do_poll`. I'll do that for all the previous occurrences as well, of course.

(It is strictly speaking controlling `sd_event_add_io` in code that is not using `poll(2)`, which is why didn't see it a namespace clash, but rather just a verb describing the action to be taken.)